src/common/util.js

```diff
@@ -0,0 +1,6 @@
+const getCompiler = (options) => {
+  if (!options.compiler) return undefined;
+  return options.compiler.split(/-|\+/)[0];
+};
+
+module.exports = { getCompiler };
```

Comment:
do you think there will be more utils? 😅

Reply:
I'm planning on moving some files from the `nodes` folder to `common` and there are some utils in the `prettier-comments/utils` that I'm preparing to clean up and move here. Since the long term goal is to delete this very outdated "dependency".